spec-main/api-tray-spec.ts

```diff
@@ -35,49 +45,45 @@ describe('tray module', () => {
     })
   })
 
-  describe('tray.popUpContextMenu', () => {
-    afterEach(() => {
-      tray.destroy()
-    })
-
-    before(function () {
-      if (process.platform !== 'win32') this.skip()
-    })
-
-    it('can be called when menu is showing', (done) => {
+  describe('tray.popUpContextMenu()', () => {
+    ifit(process.platform === 'win32')('can be called when menu is showing', function (done) {
       tray.setContextMenu(Menu.buildFromTemplate([{ label: 'Test' }]))
       setTimeout(() => {
         tray.popUpContextMenu()
+        tray.destroy()
         done()
       })
       tray.popUpContextMenu()
     })
   })
 
-  describe('tray.setImage', () => {
+  describe('tray.getBounds()', () => {
+    afterEach(() => { tray.destroy() })
```

Comment:
... here to. let's be consistent about whether or not we destroy the tray after a test.